src/components/Footer/FooterExtendedNavList/FooterExtendedNavList.tsx

```diff
@@ -14,16 +14,37 @@ type FooterExtendedNavListProps = {
 
 export const FooterExtendedNavList = ({
   className,
-  isMobile = false,
+  isMobile,
   nestedLinks,
 }: FooterExtendedNavListProps &
   React.HTMLAttributes<HTMLElement>): React.ReactElement => {
   const classes = classnames('grid-row grid-gap-4', className)
+  const isClient = window && typeof window === 'object'
+  const isMobileWidth = isClient && window.innerWidth < 480
 
+  const [isMobileFallback, setIsMobileFallback] = React.useState<boolean>(
+    isMobileWidth
+  )
   const [sectionsOpenState, setSectionsOpenState] = useState<boolean[]>(
     Array(nestedLinks.length).fill(false)
   )
 
+  // Use isMobile prop, fallback to calculated state if undefined
+  const useMobile = isMobile || (isMobile === undefined && isMobileFallback)
+
+  useEffect(() => {
+    if (!isClient) return
```

Comment:
non-blocking: just for performance, couldn't we also do an early `return` if `isMobile` is defined? since then the fallback isn't used at all and we wouldn't need to attach the event listener.

Reply:
YES.